spython/tests/testdata/docker2singularity/add.def

```diff
@@ -0,0 +1,8 @@
+Bootstrap: docker
+From: busybox:latest
+%files
+/home/vanessa/Documents/Dropbox/Code/sregistry/singularity-cli /opt
```

Comment:
Not a good idea. I'd add a marker here that the test will replace before comparison.

Reply:
I decided to not support this local path hard coding - it's a bug in Singularity that needs to be fixed.

Reply:
(The recipe testing conversion for had a "." to indicate $PWD)